### PR TITLE
Enables the `colon` SwiftLint rule

### PR DESCRIPTION
**Pull request by qsmmorgan**
The code changes were made with the `swiftlint autocorrect` command.

### CODE REVIEW

File: arcgis-ios-sdk-samples/Shared resources/Libraries/DemoTouchView/DemoTouchView.swift [hunk 1]
```diff
@@ -358,7 +358,7 @@ private class TouchView: UIView {
         
         originalCenter = center
         
-        super.init(frame: CGRect(origin: CGPoint.zero, size:size))
+        super.init(frame: CGRect(origin: CGPoint.zero, size: size))
```
Comment (philium) [inline arcgis-ios-sdk-samples/Shared resources/Libraries/DemoTouchView/DemoTouchView.swift]: Would be nice to drop the `CGPoint` here:

```suggestion
        super.init(frame: CGRect(origin: .zero, size: size))
```

File: arcgis-ios-sdk-samples/Edit data/Delete features (feature service)/DeleteFeaturesViewController.swift [hunk 2]
```diff
@@ -49,15 +49,15 @@ class DeleteFeaturesViewController: UIViewController, AGSGeoViewTouchDelegate, A
         map.operationalLayers.add(featureLayer)
     }
     
-    func showCallout(for feature:AGSFeature,at tapLocation:AGSPoint) {
+    func showCallout(for feature: AGSFeature,at tapLocation: AGSPoint) {
```
[the remark this answers is not in the document]
Comment (qsmmorgan) [reply arcgis-ios-sdk-samples/Edit data/Delete features (feature service)/DeleteFeaturesViewController.swift]: Yeah there's the `comma` rule I'll enable next.

File: arcgis-ios-sdk-samples/Edit data/Edit features (connected)/EditFeaturesOnlineViewController.swift [hunk 3]
```diff
@@ -171,12 +171,12 @@ class EditFeaturesOnlineViewController: UIViewController, AGSGeoViewTouchDelegat
     func popupsViewControllerDidFinishViewingPopups(_ popupsViewController: AGSPopupsViewController) {
         
         //dismiss the popups view controller
-        self.dismiss(animated: true, completion:nil)
+        self.dismiss(animated: true, completion: nil)
```
[the remark this answers is not in the document]
Comment (qsmmorgan) [reply arcgis-ios-sdk-samples/Edit data/Edit features (connected)/EditFeaturesOnlineViewController.swift]: Removes `nil` completion closure

**Comment by qsmmorgan** [comment]
@philium please re-approve.

**Pull request merged**